packages/fast-usdc/src/fast-usdc.start.js

```diff
@@ -72,6 +79,17 @@ const publishDisplayInfo = async (brand, { board, chainStorage }) => {
   await E(node).setValue(JSON.stringify(aux));
 };
 
+const FEED_POLICY = 'feedPolicy';
+
+/**
+ * @param {ERef<StorageNode>} node
+ * @param {FeedPolicy} policy
+ */
+const publishFeedPolicy = async (node, policy) => {
+  const feedPolicy = E(node).makeChildNode(FEED_POLICY);
+  await E(feedPolicy).setValue(JSON.stringify(policy));
```

Comment:
NTS: I've been assuming that `published` nodes have CapData but this will be raw JSON. Our client-utils should handle both